### PR TITLE
podman-compose: 0.1.5 -> 0.2.0pre-2021-05-18

### PR DESCRIPTION
**Pull request by roberth**
###### Motivation for this change

podman-compose hasn't produced a release since _over half of its commits_ (84 on Sep 2019 to 182 today), so I figured it's time for an upgrade nonetheless.

With this version, you can stop NixOS-based containers properly. I don't know what else was fixed and improved, but y'know, half of its history happened :)

> This project seems to have no release management, so the last commit is the best one until proven otherwise.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

cc @NixOS/podman @sikmir

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 123591` at 81c7dd1d run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>podman-compose</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
Near pkgs/applications/virtualization/podman-compose/default.nix:22:0:
```
   |
22 |     description = "An implementation of docker-compose with podman backend";
   | ^
```
</li>
  </ul>
</details>

File: pkgs/applications/virtualization/podman-compose/default.nix [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
+  # "This project is still under development." -- README.md
+  #
+  # As of May 2021, the latest release (0.1.5) has fewer than half of all
+  # commits. This project seems to have no release management, so the last
+  # commit is the best one until proven otherwise.
+  src = fetchFromGitHub {
+    repo = "podman-compose";
+    owner = "containers";
+    rev = "62d2024feecf312e9591cc145f49cee9c70ab4fe";
+    sha256 = "sha256:17992imkvi6129wvajsp0iz5iicfmh53i20qy2mzz17kcz30r2pp";
```
Comment (saschagrunert) [inline pkgs/applications/virtualization/podman-compose/default.nix:16]: I'm wondering what the most common default is here:
```suggestion
    sha256 = "sha256-17992imkvi6129wvajsp0iz5iicfmh53i20qy2mzz17kcz30r2pp";
```

**Review by saschagrunert — APPROVED** [review]
LGTM, just a nit from my side

File: pkgs/applications/virtualization/podman-compose/default.nix (hunk 1 above)
Comment (roberth) [reply pkgs/applications/virtualization/podman-compose/default.nix:16]: That doesn't work. `-` is for SRI hashes. This is the base32 hash you get from the got/wanted message.
I don't think we need to care about the hash format. They're all supported.

Comment (sikmir) [reply pkgs/applications/virtualization/podman-compose/default.nix:16]: The reason is that 17992imkvi6129wvajsp0iz5iicfmh53i20qy2mzz17kcz30r2pp is not SRI hash. If you would like to get proper SRI hash, use nixUnstable.

Comment (sikmir) [inline pkgs/applications/virtualization/podman-compose/default.nix:16]:
```suggestion
    sha256 = "17992imkvi6129wvajsp0iz5iicfmh53i20qy2mzz17kcz30r2pp";
```

Comment (roberth) [reply pkgs/applications/virtualization/podman-compose/default.nix]: I don't understand why you care about this, but it's done.

**Review by sikmir — APPROVED** [review]

**Pull request merged**